### PR TITLE
Requesting a pull to mozilla:master from k88hudson:t2430

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

File: make.js [hunk 1]
```diff
@@ -325,6 +325,12 @@ function buildCSS(compress) {
     cssFile: "css/embed-shell.css",
     compress: compress
   });
+
+  lessToCSS({
+    lessFile: "css/webfonts.less",
```
Comment (jbuck) [inline make.js:332]: I don't see a webfonts.less file in css, and the compiled css is checked-in

File: public/index.html [hunk 2]
```diff
@@ -1,80 +1,266 @@
...
+                The result is a whole new way of telling stories on the web—videos that are dynamic, 
+                full of links, and unique each time you watch them. It's video beyond the box. 
+              </p>
+            </div>
+            <div class="span4">
+              <video controls preload="none" width="344" height="232" 
+                poster="http://mozillapopcorn.org/wp-content/themes/popcorn/images/poster.png">
+                  <source src="http://videos.mozilla.org/serv/webmademovies/popcornsite.webm" type="video/webm">
+                  <source src="http://videos.mozilla.org/serv/webmademovies/popcornsite.mp4" type="video/mp4">
+                  <source src="http://mozillapopcorn.org/wp-content/themes/popcorn/video/popcornsite.ogv" type="video/ogg">
```
Comment (jbuck) [inline public/index.html:71]: This should be served from the CDN. If it's not available on the CDN, just nuke the ogv; we're covered by webm and mp4. Also, does mp4 need to be listed first? For some reason I remember that as a bug... maybe just in older versions of Safari or Safari Mobile.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/ad786472025259a/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/f860daee165d6de/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/ad786472025259a/output.txt

Total script time: 3.00 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/f860daee165d6de/output.txt

Total script time: 3.00 mins

File: make.js (hunk 1 above)
Comment (jbuck) [inline make.js:332]: Also, can you add linting for the HTML, CSS, and possibly JS in the public directory again?

**Comment by jbuck** [comment]
Staged on master with https://github.com/mozilla/butter/commit/76b832d6ab695f9b41413317d89aa1db66983241

**Pull request closed without merging**